c7n/filters/core.py

```diff
@@ -64,6 +64,49 @@ def operator_ni(x, y):
     return x not in y
 
 
+# intersect function will allow the comparison of 2 lists
+def intersect(x, y):
+    return bool(set(x).intersection(y))
+
+    """Filter RDS based on subnet group. Uses op intersect option to compare
```

Comment:
this doc string should probably go into the value filter docs in the docs folder for operation intersect.

Reply:
got that moved to docs.  thanks